package.json

```diff
@@ -110,7 +110,7 @@
     "@octokit/rest": "19.0.5",
     "@playwright/test": "1.26.1",
     "@types/dompurify": "2.3.3",
-    "@types/ejs": "3.1.1",
```

Comment:
Isn't this the bump PR? This doesn't look like it belongs in this PR.

Reply:
this should be gone